package.json

```diff
@@ -20,7 +20,7 @@
     "gulp-uglify": "^1.5.3",
     "knuth-shuffle": "^1.0.1",
     "testem": "^1.0.2",
-    "typescript": "^2.0.6",
+    "typescript": "^2.1.0",
```

Comment:
To make sure I understand correctly. You think we should first publish a version using TypeScript 2.0 and then afterwards one using 2.1? I don't understand the issue? Upgrading from 2.0 to 2.1 should be easy for people?